dds/DCPS/Service_Participant.cpp

```diff
@@ -787,6 +787,14 @@ Service_Participant::initialize()
   initial_SubscriberQos_.partition = initial_PartitionQosPolicy_;
   initial_SubscriberQos_.group_data = initial_GroupDataQosPolicy_;
   initial_SubscriberQos_.entity_factory = initial_EntityFactoryQosPolicy_;
+
+  initial_TypeConsistencyEnforcementQosPolicy_.kind = DDS::ALLOW_TYPE_COERCION;
+  initial_TypeConsistencyEnforcementQosPolicy_.prevent_type_widening = false;
+  initial_TypeConsistencyEnforcementQosPolicy_.ignore_sequence_bounds = true;
+  initial_TypeConsistencyEnforcementQosPolicy_.ignore_string_bounds = true;
+  initial_TypeConsistencyEnforcementQosPolicy_.ignore_member_names = false;
```

Comment:
ignore_member_names should be treated like its name suggests.  In our case, we do want to ignore the names of members for assignability so we should set it to true.

The spec has an issue regarding this setting so we should treat it as the name suggests rather than how the spec suggests until we get further clarification from OMG.

Reply:
@ClaytonCalabrese These are the defaults from the spec. I don't think we need to change them. It's meant to be used to check in the event we don't want to send the parameter.

Reply:
Son is correct I am wrong. Both the default and what we send should be false.